ui-admin/src/study/participants/enrolleeView/ParticipantNotesView.tsx

```diff
@@ -22,60 +21,46 @@ type ParticipantNotesViewProps = {
+  const [linkedTasks, setLinkedTasks] = useState<ParticipantTask[]>([])
   const { users } = useAdminUserContext()
   const sortedNotes = [...notes].sort((a, b) => b.createdAt - a.createdAt)
-  const userSelectId = useId()
 
   const { reload: reloadTasks } = useLoadingEffect(async () => {
     const tasks = await Api.fetchEnrolleeAdminTasks(studyEnvContext.portal.shortcode, studyEnvContext.study.shortcode,
       studyEnvContext.currentEnv.environmentName, enrollee.shortcode)
     setLinkedTasks(tasks)
   }, [enrollee.shortcode])
 
-  const createNote = async () => {
-    try {
-      await Api.createParticipantNote(studyEnvContext.portal.shortcode, studyEnvContext.study.shortcode,
-        studyEnvContext.currentEnv.environmentName, enrollee.shortcode, {
-          text: newNoteText, assignedAdminUserId: newNoteAssignee?.id
-        })
-      setShowAdd(false)
-      setNewNoteText('')
-    } catch (e) {
-      Store.addNotification(failureNotification('could not save note'))
-    }
-    onUpdate()
-    reloadTasks()
-  }
-
-  return <div>
-    <h3 className="h4">Notes</h3>
-    <button className="btn btn-secondary" onClick={() => setShowAdd(!showAdd)}>
-      <FontAwesomeIcon icon={faPlus}/> Add
-    </button>
-    {showAdd && <div className="pb-3">
-      <textarea rows={5} cols={80} value={newNoteText} onChange={e => setNewNoteText(e.target.value)}/>
-      <div>
-        <label htmlFor={userSelectId}>Assign to:</label>
-        <AdminUserSelect selectedUser={newNoteAssignee} setSelectedUser={setNewNoteAssignee} users={users}
-          readOnly={false} id={userSelectId}/>
+  return <div className="card w-75 border shadow-sm mb-3">
+    <div className="card-header border-bottom bg-white d-flex flex-row align-items-center">
+      <div className="d-flex align-items-center justify-content-between w-100">
+        <div className="fw-bold lead my-1">Notes</div>
+        <Button onClick={() => setShowAdd(!showAdd)}
+          variant="light" className="border m-1">
+          <FontAwesomeIcon icon={faPlus} className="fa-lg"/> Create new note
+        </Button>
       </div>
-      <div className="mt-2">
-        <button className="btn btn-primary" onClick={createNote}>Save</button>
+    </div>
+    {sortedNotes.length > 0 && sortedNotes.map((note, index) => (<div key={index}>
+      <div className='p-3'>
+        <ParticipantNoteView enrollee={enrollee} note={note}
+          studyEnvContext={studyEnvContext}
+          linkedTasks={linkedTasks}
+          reloadTasks={reloadTasks}
+          users={users}
+          key={note.id}/>
       </div>
-    </div>}
-    { sortedNotes.map(note =>
-      <ParticipantNoteView enrollee={enrollee} note={note}
-        studyEnvContext={studyEnvContext}
-        linkedTasks={linkedTasks}
-        reloadTasks={reloadTasks}
-        users={users}
-        key={note.id}/>
-    )}
+      <hr className='m-0'/>
+    </div>
+    ))}
+    { <div className='my-3'>{renderEmptyMessage(sortedNotes, 'No notes')}</div>}
```

Comment:
same nit: maybe use info card?
```suggestion
    { sortedNotes.length === 0 && <div className='my-3'>{renderEmptyMessage(sortedNotes, 'No notes')}</div>}
```